recipes/fastjet-cxx/meta.yaml

```diff
@@ -0,0 +1,134 @@
+{% set name = "fastjet-cxx" %}
+{% set version = "3.4.3" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://gitlab.com/fastjet/fastjet/-/archive/fastjet-{{ version }}/fastjet-fastjet-{{ version }}.tar.gz
+    sha256: 8ce8b9be5cd0fba4f2fe914871eabe259dfa7524edf925635e20072cde2ae6a8
+    patches:
+      - 0001-use-conda-forge-siscone-for-plugin.patch
+
+build:
+  skip: true  # [win]
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('fastjet-cxx', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ stdlib('c') }}
+    - {{ compiler('cxx') }}
+    - automake
+    - libtool
+    - make
+    - pkg-config
+    - gnuconfig
+    - cgal
+  host:
+    - siscone
+  run:
+    - siscone
+
+test:
+  source_files:
+    - example/01-basic.cc
+    - example/02-jetdef.cc
+    - example/03-plugin.cc
+    - example/04-constituents.cc
+    - example/05-eplus_eminus.cc
+    - example/06-area.cc
+    - example/07-subtraction.cc
+    - example/08-selector.cc
+    - example/09-user_info.cc
+    - example/10-subjets.cc
+    - example/11-filter.cc
+    - example/12-boosted_higgs.cc
+    - example/13-boosted_top.cc
+    - example/14-groomers.cc
+    - example/fastjet_example.cc
+    - example/fastjet_areas.cc
+    - example/data/single-event.dat
+    - example/data/single-ee-event.dat
+    - example/data/Pythia-Zp2jets-lhc-pileup-1ev.dat
+    - example/data/Pythia-dijet-ptmin100-lhc-pileup-1ev.dat
+    - example/data/HZ-event-Hmass115.dat
+    - example/data/boosted_top_event.dat
```

Comment:
`fastjet-cxx` is a large conda package at `5.0 MB`. One might be curious if removing all these `.dat` files from getting added into the `info/test/` metadata of the package does much to bring this down, but it only brings it down to `4.9 MB` so I think it is better to keep them here.